src/commands/account/get.js

```diff
@@ -0,0 +1,57 @@
+/*
+ * LiskHQ/lisk-commander
+ * Copyright © 2017–2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+import BaseCommand from '../../base';
+import getAPIClient from '../../utils/api';
+import query from '../../utils/query';
+
+export default class GetCommand extends BaseCommand {
+	async run() {
+		const { args: { addresses } } = this.parse(GetCommand);
+		const req =
+			addresses.length === 1
+				? { limit: 1, address: addresses[0] }
+				: addresses.map(address => ({
+						limit: 1,
+						address,
+					}));
+		const client = getAPIClient(this.userConfig.api);
+		const results = await query(client, 'accounts', req);
```

Comment:
I can think of three way of solving this, but it's not really good UX either...
1. Ignore errors, unless all error
2. create another `key` in the response like { data: [], errors: []}
3. print error and returned value

1 will be hard to tell what went wrong to the user, and 
considering it will be used in script, 2 and 3 will be harder to use.

any other idea to solve this? 😢

Reply:
I think the case where an address is valid but not in the blockchain is a different case to other errors. We could wrap that specific error and return an object that has no public key and no balance (therefore clear that it's never been used). With other errors it's more complicated. I guess it's more acceptable in that case to return an error if anything failed.

Reply:
In case of empty response, we are creating the error on the commander side.
but im not sure it's a good idea to `return an object that has no public key and no balance`.
Any API change will affect this command to break, while currently it's pretty flexible

Reply:
What did we do for `list` before?

Reply:
> no public key and no balance

I just meant an empty string and 0s.

Reply:
current behavior is the same as before.
if we need to initialize them, we need to be fixed on those field.
maybe what we can do is to define empty object from `key`?
for example, we call `query` with object created with key. In account case, it would be
`{account: '123L'}` and in cause of empty response, return this